src/sentry/logging/handlers.py

```diff
@@ -0,0 +1,63 @@
+"""
+sentry.logging.handlers
+~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import logging
+
+from django.utils.encoding import force_bytes
+
+
+logger = logging.getLogger('sentry.audit')
+
+
+class TuringHandler(logging.StreamHandler):
+    """
+    Decides between logging human or machine readable lines.
+    """
+    # Overridden by system.logging-format
+    fmt = 'human'
+
+    def emit(self, record):
+        context = record.args
+        # Check to make sure someone is following the rules.
+        if not context:
+            context = record.msg
+        if isinstance(context, dict):
```

Comment:
We want to make sure that even if it's not a dict, we'll go through our new emitters so that say, machine format is always machine format even if I just do `logger.info('lol')`.